.github/workflows/integration-tests.yml

```diff
@@ -241,6 +241,7 @@ jobs:
       CHAINLINK_COMMIT_SHA: ${{ github.sha }}
       CHAINLINK_ENV_USER: ${{ github.actor }}
       TEST_LOG_LEVEL: debug
+      CL_CLIENT_DEBUG: true
```

Comment:
Not sure if we need it enabled for all the tests, there will be a lot of logs.

Reply:
enabled it temporarily in order to find out the failure reason as we dont have node logs available to us. will remove it